src/sparseml/transformers/sparsification/trainer.py

```diff
@@ -725,6 +749,31 @@ def _add_tensorboard_logger_if_available(self):
             TensorBoardLogger(writer=tensorboard_callback.tb_writer)
         )
 
+    def _get_fake_dataloader(
+        self,
+        num_samples: int,
+        tokenizer: "PreTrainedTokenizerBase",  # noqa: F821
+    ):
+
+        # Rearrange inputs' keys to match those defined by model foward func, which
+        # seem to define how the order of inputs is determined in the exported model
+        forward_args_spec = inspect.getfullargspec(self.model.__class__.forward)
+        input_func = partial(
+            self._get_fake_input,
+            model_input_keys=forward_args_spec.args,
+            tokenizer=tokenizer,
+        )
+        return (input_func() for _ in range(num_samples))
+
+    def _get_fake_input(self, model_input_keys, tokenizer):
+        inputs = tokenizer(
+            "", return_tensors="pt", padding=PaddingStrategy.MAX_LENGTH.value
+        ).data  # Dict[Tensor]
```

Comment:
so will this function always return the same output which is empty string? is it useful to export 20 of the same samples in this case?